documentation/todo-app/src/App.js

```diff
@@ -1,19 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Todo from './Todo';
 class App extends Component {
+  
   render() {
+    let tasks = [
```

Comment:
A couple things here:

1. You should be using `const`, not `let`, as this array is not mutated.

2. You should declare this variable _outside_ your render function. As you have it right now, the variable is _redeclared_ each time the render function runs. This is a small issue, but would have performance issues in some cases.